_source/logzio_collections/_log-sources/azure-blob-trigger.md

```diff
@@ -0,0 +1,208 @@
+
+This integration offers the option to connect to an existing Blob storage account or create a new one, and it can be updated post-deployment.
+
+
+
+## Architecture overview
+
+The following services are created when you deploy this integration:
+
+* App Service Plan - Consumption Plan
+* Application Insights
+* Logz.io Function App + Logz.io Blob Trigger Function
+* Storage Account for Logz.io Function App logs
+
+![Integration-architecture](https://dytvr9ot2sszz.cloudfront.net/logz-docs/azure_blob/blob-trigger-resources.png)
+
+### Supported data types
+
+This Logz.io function supports the following data types:
+
+* Json
+* CSV (supported delimiters: `',' ';' '|'`)
+* Text (supports multiline text - MultilineRegex parameter)
+* The file name does not have to be with these extensions.
+
+### Supported file formats
+
+The Logz.io function supports the following file formats:
+
+* Gzip
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="new-blob-config">
+
+
+
+#### Create a new blob storage account
+
+If you don't have a general purpose v2 storage account with a container for logs, or you want to create everything from scratch, this auto-deployment is for you.
+
+* Storage Account (general purpose v2) + Container
+* App Service Plan - Consumption Plan
+* Application Insights
+* Logz.io Function App and Logz.io Blob Trigger Function
+* Storage Account for Logz.io Function App logs
+
+<div class="tasklist">
+
+##### Launch an automated deployment
+
+👇 Click this button:
+
+[![Deploy to Azure](https://dytvr9ot2sszz.cloudfront.net/logz-docs/azure_blob/deploybutton-az.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Flogzio%2Flogzio-azure-blob-trigger%2Fmain%2Fazure%2Ffull-auto-deployment.json)
+{:.override.btn-img}
+  
+##### Fill in the deployment parameters
+
+In the **Custom deployment** screen, fill in all the parameters as per table below and click **Review + create**.
+
+![Custom deployment](https://dytvr9ot2sszz.cloudfront.net/logz-docs/azure_blob/blob-trigger-1.png)
+
+| Parameter | Description |
+| --- | --- |
+| Storage Account Name | The storage account (general purpose v2) name. |
+| Storage Account Resource Name | The resource name that contains the storage account. | 
+| Container Name | The name of the container inside the storage account |
+| Logs Path | The path from where blob files will trigger the Logz.io function (including sub-directories in that path). Leave empty if you want every blob file in the container to trigger the Logz.io function. |
+| Logzio URL | {% include log-shipping/listener-var.md %} |
+| Logzio Token | {% include log-shipping/log-shipping-token.md %} |
+| Multiline Regex | The regex that matches the multiline logs in text blob files. Leave empty if you do not use multiline logs in your text blob files. |
+
```

Comment:
This is the full auto deployment section. The user will not see Storage Account Resource Name parameter (this is only for function auto deployment - this deployment will create the storage account in the same resource group so we do not need the user to fill this parameter :) )